### PR TITLE
Settings.json file added for debugging

### PR DESCRIPTION
**Pull request by Jerry-Jinfeng-Guo**
`settings.json` file added for debugging in vs code.

also follow-up to #218

### CODE REVIEW

File: .vscode/settings.json.license
Comment (mgovers) [inline .vscode/settings.json.license:1]: This license is deprecated. Please update to latest one as in literally every other file in the repo haha

Comment (Jerry-Jinfeng-Guo) [reply .vscode/settings.json.license:1]: Done. Also updated other license files (that I could find) in the repo.

File: docs/examples/data/pandapower/example_simple_input.json.license [hunk 1]
```diff
@@ -1,3 +1,3 @@
-SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
 
-SPDX-License-Identifier: MPL-2.0
+SPDX-License-Identifier: MPL-2.0
```
Comment (mgovers) [inline docs/examples/data/pandapower/example_simple_input.json.license:3]: please out of convention re-add the whiteline for all files (including the ones that didn't have it before)

Comment (Jerry-Jinfeng-Guo) [reply docs/examples/data/pandapower/example_simple_input.json.license]: Included in the latest commit.

**Review by mgovers — APPROVED** [review]

**Pull request merged**